src/components/common/AppList/AppListItem.component.tsx

```
@@ -54,6 +54,16 @@ export const AppListItem = ({
     };
 
     if (variant === 'navigation') {
+        const navItem = item as NavListItem;
+        const { name, hasDivider, icon, items: subItems } = navItem;
+        const IconComponent = iconMap[icon as keyof typeof iconMap];
+
+        const hasSubItems = navItem.items?.length ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)